setup/setup-ci/security-scanner/README.md

```diff
@@ -0,0 +1,181 @@
+
+## Getting Started
+
+> Prerequisite: an Azure ML workspace with a Compute Instance running and [diagnostic logs](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-azure-machine-learning) streaming to Log Analytics. See further down for alternative setups.
+
+- **Upload the scanner to Azure ML**: download [`amlsecscan.py`](amlsecscan.py), open [Azure ML Studio](https://ml.azure.com/), go to the Notebooks tab, and upload the file into your user folder `/Users/{user_name}` (replacing `{user_name}` with your user alias).
+- **Install the scanner**: open a terminal in Azure ML Notebooks and run `sudo ./amlsecscan.py install`
+- **Run a scan**: in the terminal, run `sudo ./amlsecscan.py scan all` (this takes a few minutes)
+
+## Assessments
+
+The security scanner installs [ClamAV](https://www.clamav.net/) to report malware and [Trivy](https://github.com/aquasecurity/trivy) to report OS and Python vulnerabilities.
+
+Security scans are scheduled via CRON jobs to run either daily around 5AM or 10 minutes after OS startup. A CRON job also emits heartbeats every 10 minutes. Scans have their CPU usage limited to 20% and are deprioritized by running at priority 19.
+
+Trivy is configured to report vulnerabilities of severity either `HIGH` or `CRITICAL` for which a fix is available. The ClamAV realtime scanning is not enabled.
+
+## Telemetry
+
+In Log Analytics, the scanner reports hearbeats to table `AmlSecurityComputeHealth_CL` and assessment results to `AmlSecurityComputeAssessments_CL`.
+
+Examples of Log Analytics [KQL](https://docs.microsoft.com/en-us/azure/data-explorer/kql-quick-reference) queries:
+- Recent heartbeats and scan status: `AmlSecurityComputeHealth_CL | top 100 by TimeGenerated desc`
+- Recent assessments: `AmlSecurityComputeAssessments_CL | top 100 by TimeGenerated desc`
+
+## Installation
+
+Irrespective of how the scanner is installed, the scanner script must first be copied to the Azure ML workspace:
+- Download [amlsecscan.py](amlsecscan.py)
+- Open [Azure ML Studio](https://ml.azure.com/)
+- Go to the Notebooks tab
+- Upload the file into your user folder `/Users/{user_name}` (replacing `{user_name}` with your user alias).
+
+The scanner can be installed on both existing and new Compute Instances.
+
+### Existing Compute Instances
+
+By default, the scanner uses the first Log Analytics workspace to which the Azure ML workspace streams [diagnostic logs](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-azure-machine-learning):
+```bash
```

Comment:
The previous text doesn't explain what this command is doing. Just that by default the scanner uses the first log analytics workspace.

Reply:
Added an explanation of what the command does.